miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -504,55 +504,55 @@ export.get_output_notes_hash
     # => [COM]
 end
 
-#! Creates a new note and returns a pointer to the memory address at which the note is stored.
+#! Creates a new note and returns the index of the note.
 #!
 #! Inputs: [tag, note_type, RECIPIENT]
-#! Outputs: [note_ptr, 0, 0, 0, 0, 0]
+#! Outputs: [note_idx, 0, 0, 0, 0, 0]
 #!
 #! tag is the tag to be included in the note.
 #! note_type is the note storage type
 #! RECIPIENT is the recipient of the note.
-#! note_ptr is the pointer to the memory address at which the note is stored.
+#! note_idx is the index of the the note.
```

Comment:
minor nit: I'd say "note_idx is the index of the crated note." here and in other similar places.